precompiles/addr/addr.go

```diff
@@ -109,6 +136,74 @@ func (p PrecompileExecutor) getEvmAddr(ctx sdk.Context, method *abi.Method, args
 	return method.Outputs.Pack(evmAddr)
 }
 
-func (PrecompileExecutor) IsTransaction(string) bool {
-	return false
+func (p PrecompileExecutor) associateWithGas(ctx sdk.Context, method *abi.Method, args []interface{}, value *big.Int) ([]byte, error) {
+	if err := pcommon.ValidateNonPayable(value); err != nil {
+		return nil, err
+	}
+
+	if err := pcommon.ValidateArgsLength(args, 4); err != nil {
+		return nil, err
+	}
+	v := args[0].(string)
```

Comment:
Nit: adding brief explanation about v,r,s could be helpful

Reply:
Added a small comment about v,r,s values here